integrations/helm/status/status.go

```diff
@@ -119,3 +119,21 @@ func UpdateReleaseStatus(client v1beta1client.HelmReleaseInterface, fhr v1beta1.
 	}
 	return err
 }
+
+func (a *Updater) UpdateReleaseRevision(fhr v1beta1.HelmRelease, revision string) error {
```

Comment:
It might be easier to do this as a function rather than a method -- that would mean you don't have to pass an extra argument to the chartsync.

Reply:
I'm not sure if it'd be easier.
If it's not a method, we will need to recreate the Updater client at inconvenient place.